README.md

```diff
@@ -1,11 +1,13 @@
-oauth2_proxy
-=================
+# oauth2_proxy
 
 A reverse proxy and static file server that provides authentication using Providers (Google, GitHub, and others)
 to validate accounts by email, domain or group.
 
-[![Build Status](https://secure.travis-ci.org/bitly/oauth2_proxy.svg?branch=master)](http://travis-ci.org/bitly/oauth2_proxy)
+**Note:** This repository was forked from [Bitly/OAuth2_Proxy](https://github.com/bitly/oauth2_proxy) on 27/11/2018.
```

Comment:
Nitpick; think this would look nicer if it was stylized as [bitly/oauth2_proxy].

Reply:
Fixed